<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotInfoPage</class>
 <widget class="QWidget" name="PlotInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1">
     <item row="1" column="0">
      <widget class="QFrame" name="frameOp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnNewAddress">
          <property name="text">
           <string>New Plot ID</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbNewAddress">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>New Mining Address</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formNew">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Miner Address:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="ebNewAddress"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="plotIdLabel_2">
               <property name="text">
                <string>PlotId:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="ebNewId"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="currentInfoFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Current Mining Info:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="minerAddressLabel">
            <property name="text">
             <string>Miner Address:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ebMinerAddress">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="plotIdLabel">
            <property name="text">
             <string>PlotId:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ebPlotId">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Bind Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ebAddressFrom">
         <property name="placeholderText">
          <string>Binding From Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Bind To Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ebAddressTo">
         <property name="placeholderText">
          <string>Binding To Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBind">
         <property name="text">
          <string>Bind PlotId</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnbind">
         <property name="text">
          <string>UnBind PlotId</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Binding Info:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ebAddrToQuery">
         <property name="placeholderText">
          <string>Address To Query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuery">
         <property name="text">
          <string>Query Binding</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ebMinerAddress</tabstop>
  <tabstop>ebPlotId</tabstop>
  <tabstop>btnNewAddress</tabstop>
  <tabstop>ebAddrToQuery</tabstop>
  <tabstop>btnQuery</tabstop>
  <tabstop>ebAddressFrom</tabstop>
  <tabstop>ebAddressTo</tabstop>
  <tabstop>btnBind</tabstop>
  <tabstop>ebNewAddress</tabstop>
  <tabstop>ebNewId</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
